ddtrace/tracer/span.go

```diff
@@ -66,6 +66,7 @@ type span struct {
 	TraceID  uint64             `msg:"trace_id"`          // identifier of the root span
 	ParentID uint64             `msg:"parent_id"`         // identifier of the span's direct parent
 	Error    int32              `msg:"error"`             // error status of the span; 0 means no errors
+	errCfg   errorConfig        `msg:"-"`                 // configuration for errors tracing
```

Comment:
Should we just pass the whole config pointer? Does it make sense? Fine as is too.

Reply:
You mean having a `FinishConfig` rather than an `errorConfig`? 
That's a fine idea, and we can do more with options in the future if we want.

Reply:
Hmm... I ended up removing both `FinishConfig` and `errorConfig`. We were only using one field from each, so it seemed counter-intuitive to include them. 

I still think it could be a good idea, but we can follow up in another issue/PR.

Reply:
Lgtm